pom.xml

```diff
@@ -706,6 +797,36 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>java8-test-profile</id>
+            <activation>
+                <property>
+                    <name>java8.home</name>
+                </property>
+            </activation>
```

Comment:
This needs to be added to work with Java 8
```
<properties>
    <modular.jdk.args/>
    <modular.jdk.props/>
</properties>
```

Reply:
Added it